js/script.js

```diff
@@ -0,0 +1,74 @@
+let now = new Date();
+let year = now.getFullYear();
```

Comment:
You are not changing these variables, so use const
```
const now = ...
const year = ...

// It can be in one line
const year = (new Date()).getFullYear();
```